node-fs-writefile/cp.js

```diff
@@ -0,0 +1,13 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function cp() {
+  try {
+    const contents = await readFile(process.argv[2], 'utf8');
```

Comment:
By using 'utf8' you make this so it will only copy text files.

Reply:
Removed utf8